### DISCUSSION TITLE
Changing "Unlisted" (PR #28119)

### DISCUSSION THREAD

**Discussion by renchap** [mastodon/mastodon#28887] [General]
This discussion is extracted from https://github.com/mastodon/mastodon/pull/28119 to specifically discuss the "Unlisted" renaming that is proposed in this PR. Also note that I have not been personally involved in this, I am trying to summarise what I have been able to gather on the topic.

First of all, please keep this discussion constructive and friendly.

For context, Mastodon supports 4 status visibility levels, including "Unlisted".

It is currently displayed as follows in the UI:

> Unlisted: Visible for all, but opted-out of discovery features (open lock icon)

An unlisted status is a public status, but:
- does not appear in hashtag feeds
- does not appear in the federated feed
- does not appear in the local feed
- can not appear in trends
- can not appear in search

This visibility level has always created issues and misunderstandings, and we get a lot of feedback and questions about it.

Major issues are:

- one of the only other social places with "Unlisted" vocabulary is Youtube, but it does not has the same meaning as Mastodon (unlisted videos on Youtube are not discoverable nor displayed in any listings)
- it does not make it obvious that the status will still appear on your profile or on your follower's timelines
- it tends to make people think this will make their status more private that what it will really be

After some brainstorming with some people in the team (but not an agreement from everybody), the PR has it changed to
> Low-key public: Public with less reach (half-moon icon)

The intention is to have "public" in its name, to better convey that this is a variation of the public visibility, but will be less visible.

Another discussed alternative is having it as a toggle under the "public" visibility setting, so it would be "public but without discoverability". This requires significant UI changes in the dropdown component and has not been done in this PR, but might be the "correct" solution. It also does not solve how to convey this when displaying a status (the existing lock icon).

I hope this helps providing some context for the change, and if you have any ideas on how to solve the issues exposed above, let's discuss about it and see if we manage to find a better way :)

**Comment by Andre601** [comment]
Just to understand it right, unlisted doesn't post it to the public (federated) timeline. Will it still be on the instance (local) one?
If so, why not call it "local"?

**Reply by trwnh** [reply]
It will not.

Unlisted opts out of all Mastodon-specific discovery features, including:

- The public timelines
  - Federated
  - Local
  - Hashtag
- Full-text search

**Reply by Andre601** [reply]
Then what about "Personal"? Since it still shows up on your personal timeline?

Maybe with a description such as "Post shows in your personal timeline, but is hidden from any form of discovery"

That "Low-key public" just sounds way too out of place here...

**Reply by renchap** [reply]
Unlisted posts are also not allowed to trend.

I updated the initial post adding a description of how those posts work.

**Comment by tsmethurst** [comment]
I will link to my review comment here: https://github.com/mastodon/mastodon/pull/28119#issuecomment-1908142688

I'd also add that since a user's choice between public and unlisted affects how the Note is addressed when it federates out (public uri in To vs public uri in CC), it should also be made clear to the user that their visibility-level decision also affects how AP implementations other than Mastodon will treat the post.

It's also worth bearing in mind that many third party apps already use the "unlisted" terminology, and many users are already used to that terminology, so changing it now (especially to something vague like "low-key public") is liable to create more confusion than it resolves.

I believe the best way to convey the meaning of unlisted to a user is to keep "unlisted" as a term, make sure "unlisted" is used consistently across the web UI and across first-party mobile apps, and to document "unlisted" properly, linking the user to documentation if possible.

As an aside, I also think it's procedurally very odd, if Mastodon actually wants to gather feedback from github users in advance of a change, for the change to be bundled into a PR without external discussion, and then to only create a venue for external discussion when the change is noticed. In my view this serves to create the illusion of discussion when the momentum is already there (the code is already written) for the change to be pushed through regardless. If it's already a done deal, then fair enough, that's fine, but there's not much point proceeding with the trappings of gathering external feedback in that case.

Anyway, best of luck with the discussion, I hope you can find a way forward that works well!

**Comment by trwnh** [comment]
Suggestion:

- Rename "unlisted" to "public" (because it is literally that, and federates as Public)
- Rename "public" to "discoverable"
  - #12178 

State afterward:

- Discoverable
  - "Included in discovery features"
  - Icon: Globe with magnifying glass https://fonts.google.com/icons?selected=Material+Icons:travel_explore
- Public
  - "Visible to anyone"
  - Icon: Globe https://fonts.google.com/icons?selected=Material+Icons:public
- Your followers
  - "Send to your followers, plus anyone mentioned"
  - Icon: Lock with a plus https://fonts.google.com/icons?selected=Material+Icons:enhanced_encryption
- Specific people
  - "Everyone mentioned can see this"
  - Icon: At-sign https://fonts.google.com/icons?selected=Material+Icons:alternate_email

**Comment by mkody** [comment]
It might be interesting to share that Misskey uses "Home - Post to home timeline only".
![image](https://github.com/mastodon/mastodon/assets/843787/69aedb8f-3247-4f24-ac27-a7123611e26f)

**Reply by ClearlyClaire** [reply]
Does it work the same way? e.g. does it show up on your profile? Because “unlisted” does, and the wording of “Home” as described there would imply it doesn't.

**Reply by mkody** [reply]
It does work the same. The unlisted posts from Mastodon appear has Home on Misskey, and the same the other way around, and they're visible on my profile even logged out.
The short description should be "home timeline and your public profile" (maybe they're keeping it short or not considering clearing that up)

**Reply by mkody** [reply]
Adding also that posts using it does not appear in the "Local" timeline (that's an another option when clicking "defederate" with a public post to make it local-only).
Using "Home" follows the name of the "main" timeline, which not only includes the people you follow but also content that has been boosted by them, as long as it was not mentioned/followers-only.

**Reply by LuNeder** [reply]
If we really have to change from “unlisted”, this is definitely the best option anyone has mentioned

**Reply by brendanjones** [reply]
> does it show up on your profile? Because “unlisted” does, and the wording of “Home” as described there would imply it doesn't.

@ClearlyClaire I don't agree. No matter which timelines a post goes to, it's always going to display on your profile to anyone with the viewing privileges to see the post. Your profile is not a timeline, it's a collection of all your posts and replies. Like, technically yes your posts will appear there chronologically sorted so one could argue it's a timeline ... but it's not a timeline, it's a profile. Post visibility shouldn't affect which posts go there, they all go there.

**Reply by brendanjones** [reply]
'Home' is my favourite change, because it clearly explains where the post goes. Home timeline only. 

There's only one asterisk: iiuc unlisted posts also go to lists, so more accurate would be "Post to home timeline and lists only".

**Reply by LuNeder** [reply]
They appearing on lists is kinda expected tho? Like, lists are pretty much alternative home timelines.
Don’t private toots show up on lists too (if you follow who tooted them)?

**Reply by brendanjones** [reply]
Expected by who? I can assure you, many people don't understand what posts go where, including which go to lists. For example, check out how many people don't know what unlisted does, based on a [poll I ran](https://fosstodon.org/@Brendanjones/110894499551248739). I suspect there's similar ignorance about what 'public timelines' means.

That said, in order to keep the copy clean I would be fine with the shorter "Post to home timeline only" *as long as* it's very clearly stated in the Mastodon documentation that these posts do go to lists: https://docs.joinmastodon.org/user/posting/#privacy. 

Those docs need an update to more clearly explain post distribution/visibility, and there should be a link to them from post distribution settings in the post composer. << @renchap it'd be good if this was done no matter what changes are/aren't made to 'unlisted'.

**Reply by Gargron** [reply]
I would not use "Home" because that might as well be the "Followers-only" option, because followers obviously see you in their home feed. The difference between them would be very hard to understand and confuse people just as well.

**Reply by brendanjones** [reply]
> I would not use "Home" because that might as well be the "Followers-only" option, because followers obviously see you in their home feed.

Sorry @Gargron but this argument can be made for any name we give Unlisted, because Unlisted and Followers Only are very similar in how they work (at a high level. I do understand how they're different). Unless you put way too many details in the post composer UI then there's no way to explain exactly what each visibility setting does. That's why https://docs.joinmastodon.org/user/posting/#privacy should be linked.

> The difference between them would be very hard to understand and confuse people just as well.

This is exactly my problem with "Low-key public: Public with less reach". At least "Home: Post to home timeline only" describes where the post *is* going to go. "Low-key public" tells me nothing.

**Comment by brendanjones** [comment]
> Low-key public: Public with less reach

Whatever happens, better to stick with Unlisted than change to this. It's even more confusing than Unlisted.

**Reply by brendanjones** [reply]
There has been a *lot* of discussion around the problems with Unlisted already, and how to fix it. Good to have a read before weighing in on this PR. Linking the issues I'm aware of:

- https://github.com/mastodon/mastodon/issues/5463
- https://github.com/mastodon/mastodon/issues/12178
- https://github.com/mastodon/mastodon/issues/21245
- https://github.com/mastodon/mastodon/issues/21645
- https://github.com/mastodon/mastodon/issues/23038
- https://github.com/mastodon/mastodon/issues/26864

Please link any others I've missed.

**Comment by msikma** [comment]
> one of the only other social places with "Unlisted" vocabulary is Youtube, but it does not has the same meaning as Mastodon (unlisted videos on Youtube are not discoverable nor displayed in any listings)

On Youtube, if you put an unlisted video in a public playlist, those videos are visible to all users in the playlist. For example, [all the videos here are unlisted](https://www.youtube.com/playlist?list=PLHZ-lw26q81UpyWG1td2Aha2IrkEhlCXX). Still, not entirely the same as Mastodon. I don't believe there are any other ways to discover unlisted videos other than through inclusion in a public playlist.

I understand that "unlisted" is not entirely obvious, but "low-key public" sounds like a really confusing term to me.

It's also very difficult to understand unless you're a native English speaker. And I know some people will say that if you're not a native English speaker, you should just be using the Spanish or Dutch or German interface or whatever, but in practice non native speakers interact with English systems every day. I don't think it's bad to avoid the use of slang, even if it's pretty well established slang, if it's not strictly necessary.

Again, I understand that "unlisted" is confusing because it's not truly unlisted, but if you merge this then you will get even more confused users than you had before.

Part of the problem is that it's not really "private" and it's also not really "unlisted". Unlisted posts are actually listed: they're on your followers' timelines and on your profile. I can't off the top of my head think of any parallels to that in other social media sites. So I don't know what prior art exists.

**Comment by Akselmo** [comment]
I am not native english speaker. What does "lowkey public" mean?

Unlisted makes more sense.

PS. *Yes, I do know what it means. But I wouldn't know it if I wasn't terminally online. This wording translates really badly as well.*

**Comment by Gargron** [comment]
I am changing "Unlisted" to "Low-key public" because "Unlisted" is wrong. There are a lot of very common misconceptions about what it does because of the assumptions people make from the word "unlisted". The nature of the option itself is incredibly vague and requires a lot of external information to understand fully. It was originally a moderation feature for punishing spammers, and I have long regretted agreeing to making it a user-facing option because of the confusion it causes and how badly it fits within the gradient of other options. The truth is that for most intents and purposes, "unlisted" is "public", except it makes less "noise". Hence the half-moon symbol, commonly used for "silent mode". What does making less noise mean? Well, what does "opted out of discovery features" mean? It is less public, but still public, and to find out exactly how less public it is, you'd need a long list of caveats[^1]. After a number of discussions with members of the community and our designer about possible terminology, "low-key public" is what I've got. Am I concerned that "low-key" is slang? Yes. But it actually encapsulates the nature of this feature a lot better than any more formal phrase. And if according to some having "toot" in the interface was okay, then "low-key" can be okay as well.

[^1]: Can be viewed by logged out users, can be viewed on profile, can be viewed via link, does appear in followers' home feeds; does not appear in live feeds, does not appear in hashtags, does not appear on explore, does not appear in search on Mastodon (but may appear in web search e.g. Google)

**Reply by tsmethurst** [reply]
> I am changing "Unlisted" to "Low-key public" because "Unlisted" is wrong.

Well there you go then. If the decision is made already, then framing any of this as a discussion seems a rather pointless PR exercise, as I alluded to in my comment above (https://github.com/mastodon/mastodon/discussions/28887#discussioncomment-8236633).

As an aside, can I recommend that Mastodon considers setting its Github repo to "contributors only" for comments and pull requests, to avoid giving the impression that unsolicited feedback from external sources is welcomed? That would lead to far fewer misunderstandings of this nature.

**Reply by YJSoft** [reply]
As non native English speaker, I don't understand what is "low-key" is. Only means I can think is "key is placed at lower places"

Also, "low-key" cannot be translated into Korean easily since there are no matching word. I bet some other languages will have same problem too.

So that change only increases confusion, rather than solving it.

> And if according to some having "toot" in the interface was okay, then "low-key" can be okay as well.

No. users don't need to understand what toot is to write it, but users might need to understand what is "low-key" is to write as it.

**Reply by Gargron** [reply]
There is a number of [synonyms](https://www.thesaurus.com/browse/low-key) that can be used for translations.

**Reply by Akselmo** [reply]
What about "semi-public?"

"Quiet" could also work, as in talking quietly: not everyone but those nearby hear it.

This could also be changed all around:

- shout (public: everyone sees this on all feeds)
- talk/quiet (unlisted)
- whisper (only followers)
- direct messages, this works on its own

**Reply by FlitPix** [reply]
> As an aside, can I recommend that Mastodon considers setting its Github repo to "contributors only" for comments and pull requests, to avoid giving the impression that unsolicited feedback from external sources is welcomed?

@tsmethurst This would be fine, if the repo were not the _only_ place to give feedback, and if large accounts like Fedi.Tips were not already linking to issues on the repo for users to give feedback.

Setting up a proper, user-oriented space for feedback is 100% a prerequisite to this - otherwise, we're leaving users with no place to leave direct feedback, aside from flooding the official Mastodon account with mentions.

**Reply by yeenbean** [reply]
If my mom were to join Mastodon, she'd have no idea what "low-key public" would mean. You're using language that is used by a small audience of your software, which restricts accessibility to the platform. Despite finding the humor in this change on a personal level, this is ultimately a **bad** change overall and will cause severe confusion amongst users—especially people who are new to Mastodon and federated social networking.

**Reply by Kiloku** [reply]
> > As an aside, can I recommend that Mastodon considers setting its Github repo to "contributors only" for comments and pull requests, to avoid giving the impression that unsolicited feedback from external sources is welcomed?
> 
> @tsmethurst This would be fine, if the repo were not the _only_ place to give feedback, and if large accounts like Fedi.Tips were not already linking to issues on the repo for users to give feedback.
> 
> Setting up a proper, user-oriented space for feedback is 100% a prerequisite to this - otherwise, we're leaving users with no place to leave direct feedback, aside from flooding the official Mastodon account with mentions.

The point is that Gargron and the Mastodon project in general clearly don't want feedback, his decisions are final and even though it's an open source project, we're at his whims because mastodon.social is more influential than all other instances and even all other ActivityPub projects. This situation made that extremely clear. 

Hopefully a truly community oriented ActivityPub project takes over as the "main" fedi platform at some point.

**Comment by LuNeder** [comment]
>Another discussed alternative is having it as a toggle under the "public" visibility setting, so it would be "public but without discoverability". This requires significant UI changes in the dropdown component and has not been done in this PR, but might be the "correct" solution. 

Considering the concerns, I agree that this would be the most correct solution. When it’s put at this way, unlisted isn’t really a toot privacy like the other ones but indeed a discoverability setting of public.   
In the meantime, I’d say the best is to keep it called “unlisted”. Unlisted might not convey the 100% correct meaning, but it does it way better than “low-key public”. The “low-key public” labels means absolutely nothing and, honestly, sounds like a joke. I actually high-key thought the screenshot was a meme when I saw someone’s toot with it, and I was not the only one.

If this is such an urgent problem that delaying it and changing the UI significantly is an issue, then I’d say something closer to the “Home” solution by mkody is the second best choice. Actually describing what will happen with something you will toot is important, and “public with less reach” describes nothing at all.

> It also does not solve how to convey this when displaying a status (the existing lock icon).

I think most of us here are disagreeing with the “low-key” naming and absolutely non-descriptive description that follows it on that PR. The half moon choice isn’t bad, and a ghost or a lil hiding emoji wouldn’t be bad choices either.

**Comment by temtemy** [comment]
I don't like "Low-key public" as it's too wordy IMO, and can be a problem in l10n. If renaming to "Home" like what Misskey uses is not an option, then I suggest "Subtle", because that's usually my intent when I set my post's visibility to Unlisted anyway. I'd want my post to still be viewable from my profile even when logged-out, but I don't want it to be secret (as in followers-only which will be invisible to logged-out users), nor discovered by full-text search or hashtag.

With "Subtle" the crescent moon would make much more sense now, I think.

**Comment by jesse-tong** [comment]
I think even though "Unlisted" is not perfect, it's still better than "Low-key public" for me. An average user, especially non-native English speakers, would ask what is "low-key" and how it is considered like that. Though given that unlisted posts are not indexed for hashtag and full-text searches, and also not counted or index for trending or public/federated timelines, I suggest "Unindexed" as an replacement though.

**Comment by quillcraftsman** [comment]
I think "Public with less reach" this part is good and "half moon icon" is perfect. But I agree that "Low-key public" sounds a little bit strange. 
May be something like "Restricted Public" or "Restricted Reach". 
Or may be to keep "Unlisted" and to change the description and icon: ( Unlisted: Public with less reach (half-moon icon)  ) in this way old users won't be confused and new users will be understand more about this status.

**Comment by rscmbbng** [comment]
Considering it is about reducing visibility and amplification, unamplified public would be my suggestion.

**Comment by MadokVaur** [comment]
I've followed this issue here in both this and the previous, parent Issue, "Change design of compose form in web UI #28119"

All of this has now surfaced on Mastodon itself

The question asked there was essentially "What does this do to the tens of thousands of people who already read and understand the context and meaning of the current wording? Why and for what, will they be forced to relearn new terms for existing functions?" 

Then, as to "low-key public" -- low-key actually has a listing at the Urban Dictionary

It's best described as edgy Anglo teen-speak, taken (as is so often the  case) completely out of context from the meaning found at Websters Dictionary

The example I used on Mastodon from the gamer universe: "Imma low-key gonna get me some Chipotle while I pop some caps in Call of Duty on Twitch"

Is that the cultural context Mastodon wants to put forward to the world at large with this significant edit?

Edgy teen-speak?

**Comment by renchap** [comment]
Small update here: the composer rework PR has been merged as we want to be able to test it widely (big change, lot of potential issues).

Based on the feedback here and elsewhere, and some good suggestions, it has been changed to this:

![](https://files.mastodon.social/cache/media_attachments/files/111/817/211/330/409/048/original/978653af6b444565.png)

There might be other iterations on this (for example replacing the tooltip as it might not work well on some devices at the moment) and we are still listening to feedback, but this will be done in further PRs.

**Reply by brendanjones** [reply]
Extremely glad you've chose for Quiet Public over Low-key Public. I'm not sure about the description, though, because that raises more questions than it answers. What is an algorithmic fanfare, and what algorithmic fanfares happen when a post is Public? Because this implies that posting Public causes algorithmic fanfares.

Might I suggest something simpler like "Quiet Public: Appears in fewer places"?

**Reply by LuNeder** [reply]
That’s way better, thanks for listening!

The little (i) description is great!
Tho I’m not sure if “algorithmic” is the best way to describe it in the mini description since the local and federated timelines don’t really fit that?

**Reply by brendanjones** [reply]
Also, 'live feeds' seems to be a new term. What is a live feed? Aren't all feeds live feeds? Is this the new term for public feeds?

Why not be specific and say:
> This behaves exactly like public, except the post will not appear in federated, local or hashtag feeds, explore, or Mastodon search, even if you are opted-in account-wide.

**Reply by andypiper** [reply]
"live feeds" has been an item in the logged-in web UI sidebar since Mastodon 4.2, and links to a page that provides "this server", "other servers" and "all" feed views.

**Reply by mcclure** [reply]
I personally don't like this new wording. However I think the (i) is a more prominent problem in that it appears to do nothing at all. I was actually partway through composing a "New compose box has a (?) which does nothing" issue when I retested and discovered I just wasn't hovering over the (i) long enough for a tooltip to appear.

In general, proper tooltips in web apps tend to appear much quicker than the "image caption"/"title" tooltips the browser inserts for an image. Since the ⓘ is not obviously an image, I think an average user's expectations would be that mousing over or clicking the ⓘ should be enough to get any help message to appear, especially since the UI *does* respond immediately in the form of changing the cursor to a question mark. (I would not usually understand that when the cursor changes to a question mark, the message the app is sending me is "now hold your cursor totally still for several seconds".) If the message doesn't appear, and the user moves the mouse or clicks before the browser displays the title tip (as happened to me in repeated tests) this will appear to be broken.

**Reply by lapcat** [reply]
> I think the (i) is a more prominent problem in that it appears to do nothing at all. 

Agreed. When I first saw this, I tried to hover over the (i), then click it, in order to get more information, and was confused that it did nothing.

**Reply by renchap** [reply]
As I said in my post above, we intend to replace this tooltip with something else, this was a quick implementation to get it merged.

**Reply by mcclure** [reply]
I guess this isn't a very helpful comment, but it seems like this entire chain of changes (replace images with SVGs, introducing visual ambiguity on icons; replace visual ambiguity with new Compose design which itself is confusing without to-be-implemented features) has comprised the devs repeatedly rushing incomplete changes into production to correct the issues caused by the previous incomplete change.

**Reply by Akselmo** [reply]
This is way better, cheers!

**Comment by pionaiki** [comment]
How about "non-discoverable"?

**Comment by lapcat** [comment]
I just got "Quiet Public" on my mastodon.social home, and I find it very confusing and unhelpful. Unlisted was much clearer in meaning to me. Quiet Public is almost an oxymoron, and "Fewer algorithmic fanfares" sounds too cutsey while not saying what it really means, my understanding of which is that the post is not indexed for search.

**Comment by jazmichaelking** [comment]
A confounding issue that I think is adding to the complexity is that the terms are confused as to whether they are nouns or adjectives in this context. 

In the question of visibility (not privacy, those are account settings) in response to "who can see this?" we are thinking nounally to describe "the public", "only the people who follow me", "only the people I mentioned", and would therefore require a noun for "unlisted", but "public" and "unlisted" in this context are adjectival,  which I believe is causing the bump in the road. Once we modify "public" we are now thinking of "public" in adjective form. However, there is no solid option for a noun that means "visible to public but only in certain ways" as we've now left the noun station and are very much on our way to adjective town.

We could use a superlative for "public", which would be "most public", but that doesn't sit well and ideally we (I imagine) want "public" to be the top-level, most used visibility choice as a default; we don't want to make the current "public" superlative as that in itself adds confusion.

My guess is we squeeze an adjective under "Public" as has been suggested, but go with a widely recognised adjectival approach. 

"Semi-public" has no slangy overtones, clearly modifies "public" both in and out of UI context; is adjectival in and of itself ("quiet" in this usage should be the adverb "quietly" as we are modifying an adjective -> out of context from the preceding "public", "quiet public" means the body of people, the noun, is quiet - as in "a public that is quiet"); and "semi" is a common prefix that is widely understood and used by non-native speakers (e.g semi-skim, semi-colon, "semi-final"). 

The only other options I can think of are "partial" which can be confusing due to its second meaning as being in favour of one side over another, and "quasi" which is a less common and harder-to-read prefix. 

https://dictionary.cambridge.org/us/dictionary/english/semi-public

Re; "unlisted": "Unlisted" is dichotomous, it is the opposite of "Listed" and has a finality and definitiveness to it. It does not infer "less than", it infers "not". Additionally, "unlisted" in English-speaking countries is more often comprehended in the context of being unlisted in directories (telephone numbers, stock security offerings), it is a heavy dose of privacy that "unlisted post" does not actually confer. By modifying "public" with something that signals "less than public" as opposed to "not public", we can negate the unintended comprehension that "unlisted" is more private than it truly is. 

As an aside, I'm not certain I fully grasp the interplay of the account privacy search settings and individual post visibility settings.

**Comment by ianh** [comment]
Throwing a suggestion into the hat: change unlisted to “public”, then change public to “discoverable” to communicate the difference between the two.

**Comment by shleeable** [comment]
"Everyone on and off mastodon" needs to be replaced.

**Comment by todoneunl** [comment]
I have a solution I think: 

**Less public**

**Reply by todoneunl** [reply]
Or as earlier mentioned:

**Semi-public**

**Comment by shleeable** [comment]
Low-key is cute... but for something that is used as "limited visibility", I'd prefer the more human "**Not Promoted**".

semi-public / selective public / limited broadcast / controlled public / discreet public / curated public... different kinds of vague

**Comment by slrgt** [comment]
Wouldnt "less public" make more sense than "low-key public"?

**Reply by daprice** [reply]
But it isn’t less public at all. “Less public” would imply a restriction on who is allowed to see it. It’s just as public as ever, it just isn’t promoted.

“Less public” is also nonspecific and just makes you wonder “how much less public?”

**Reply by todoneunl** [reply]
It is less public, because it is not visible on public timelines, hashtags and trends. Still public on users home timelines, lists and profiles. So that's less public.

**Reply by daprice** [reply]
If you go on national TV and say something, you’re saying it in public.

If you go to a street corner and say it where anyone can hear, but there aren’t TV cameras broadcasting it, that’s not less public.

**Reply by todoneunl** [reply]
I think it does. If I say something awkward at a street corner, I'm much more relieved when it's broadcasted on national TV.

**Reply by brendanjones** [reply]
Guys there's not much point discussing whether less-public is better than low-key public when they've already gone with quiet public over low-key public See: https://github.com/mastodon/mastodon/discussions/28887#discussioncomment-8247869

**Comment by daprice** [comment]
I hate to be pedantic, but I don’t think “fewer algorithmic fanfares” is grammatically correct. “Fanfare”, as in the abstract definition meaning publicity, is an uncountable noun, so it should be “less algorithmic fanfare”. Using it as a countable noun (“a fanfare” or “fanfares”) would mean you’re talking literally about music being played on a trumpet.

[Merriam-Webster](https://www.merriam-webster.com/dictionary/fanfare) has examples with “more fanfare” and “less fanfare”.

I’m not a fan of this name or explanatory text in the first place (it has no meaning to anyone who isn’t a Mastodon algorithm developer) but if it’s going to be kept, it should be grammatical.

**Reply by brendanjones** [reply]
I took it to mean the countable version, as in each notification triggered or whatever counts as one fanfare. In which case 'fewer' is correct.

Not that I really care either way, the phrase is not great, as I commented [above](https://github.com/mastodon/mastodon/discussions/28887#discussioncomment-8248468).

**Comment by quillcraftsman** [comment]
Today I added some Russian translations on Crowdin. I just want to notice that sometimes it wasn't easy. For example "Fewer algorithmic fanfares" - I know how it be in Russian but I don't know what it means in Russian. There is a same comment about Dutch on Crowdin.

**Comment by darltrash** [comment]
I am not exactly well versed with this but, what if instead of changing only this one entry, we straight up changed what the button means in general?  As in:

Discoverability:
- The entire world
- This instance
- This profile
- Tagged
